crates/ethereum/evm/src/lib.rs

```diff
@@ -155,10 +170,9 @@ mod tests {
 
         // Use the `EthEvmConfig` to fill the `cfg_env` and `block_env` based on the ChainSpec,
         // Header, and total difficulty
-        EthEvmConfig::default().fill_cfg_and_block_env(
+        EthEvmConfig::new(MAINNET.clone()).fill_cfg_and_block_env(
```

Comment:
we should reuse chainspec defined above here

Reply:
done